src/core/reactor.cc

```diff
@@ -226,6 +232,12 @@ struct convert<seastar::mountpoint_params> {
         if (node["rate_factor"]) {
             mp.rate_factor = node["rate_factor"].as<float>();
         }
+        if (node["discard_block_size"]) {
+            mp.discard_block_size = parse_memory_size(node["discard_block_size"].as<std::string>());
+        }
+        if (node["discard_requests_per_second"]) {
```

Comment:
How to calculate this number for a given node?

Reply:
It is a quite tricky question. The experiments with `diskplorer` seemed to indicate, that when `block_size == 32MiB` and `discard_bw == write_bw`, then on i3.2xlarge such huge latency spikes had not been visible.

However, what I am afraid of is whether the configured bandwidth would be sufficient. The queue of files to remove cannot grow indefinitely. Also, I am not sure how to measure the desired value via iotune - creation of files to test certain bandwidth may take long time.